src/preview.js

```diff
@@ -57,17 +87,7 @@ const getPreviewHeader = ( lang, imageUrl = '' ) =>{
 		return `
 			<div class="wikipediapreview expanded ${isTouch ? 'mobile' : ''}" lang="${lang}" dir="${dir}">
```

Comment:
This line is still duplicated in all entry points. It could be extracted into some kind of `render( lang, isTouch, headerContent, bodyContent, ctaContent )` that handles the general container for all states.